example_backend_profile/settings.py

```diff
@@ -1,13 +1,24 @@
 from pathlib import Path
+import environ
 
 # Build paths inside the project like this: BASE_DIR / 'subdir'.
 BASE_DIR = Path(__file__).resolve().parent.parent
 
-# SECURITY WARNING: keep the secret key used in production secret!
```

Comment:
We could still paste this warning to every possible location just in case...

Reply:
Meaning, also to config.env.example

Reply:
This text was moved downwards in this file a bit. But since `settings.py` isn't the place where one sets the password anymore, this warning is a bit misplaced and a better place for it would be the `config.env.example` file. So you're absolutely right.